material-stepper/src/main/java/com/stepstone/stepper/StepperLayout.java

```diff
@@ -564,6 +566,14 @@ public boolean isInProgress() {
     }
 
     /**
+     * Sets layout direction of mPager
+     * @param isRtlEnabled true if mPager should be swiped from right to left
+     */
+    public void setRtlEnabled(boolean isRtlEnabled) {
+        this.mIsRtlEnabled = isRtlEnabled;
```

Comment:
Done, your recommendations are implemented in the last commit.

By the way, I didn't know that `value` folder have RTL functionality which really simplifies RTL implementation greatly.